app/Observers/AppointmentObserver.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Observers;
+
+use App\Models\Appointment;
+use App\Services\DoctorService;
+use App\Services\FirebaseService;
+use App\Services\PatientService;
+use Illuminate\Http\Response;
+
+class AppointmentObserver
+{
+    /**
+     * Handle the Appointment "created" event.
+     *
+     * @param  \App\Models\Appointment  $appointment
+     * @return void
+     */
+    public function created(Appointment $appointment)
+    {
+        $doctor = app(DoctorService::class)->show($appointment->doctor_id);
+        if ($doctor->firebaseTokens()->count() > 0) {
+            $tokens = $doctor->firebaseTokens()->pluck('token')->toArray();
+            app(FirebaseService::class)->pushNotification($tokens);
```

Comment:
Please specify the notification title and body for both doctor case and patient case